torch/_inductor/codecache.py

```diff
@@ -1973,7 +1973,7 @@ def _compile_consts_linux(consts: bytes) -> str:
                 run_command_and_check(cmd)
             log.debug("aot constant binary command: %s", cmd)
 
-            if graph.buffer_mutation:
+            if graph.mutated_buffers | set(graph.constants.keys()):
```

Comment:
wouldn't this be much more efficient (O(1) vs O(N)) if written `if graph.mutated_buffers or graph.constants:`?

Reply:
Actually I got this wrong, it should be `&`. Fixed.